lib/cdo/session.rb

```diff
@@ -0,0 +1,6 @@
+# Environment-specific session cookie key constant.
+require_relative '../../deployment'
+class Session
+  KEY = "_learn_session#{"_#{rack_env}" unless rack_env?(:production)}"
```

Comment:
The doubly nested #{ #{ } #} are hard to read and repeated on two lines.

Could you change this to:
env_suffix = "_#{rack_env}" unless rack_env?(:production)
KEY = "_learn_session#{env_suffix}"
STORAGE_ID = "storage_id#{env_suffix}"